sub/sd_ass.c

```diff
@@ -831,6 +840,7 @@ static int control(struct sd *sd, enum sd_ctrl cmd, void *arg)
             assobjects_destroy(sd);
             assobjects_init(sd);
         }
+        ctx->ass_configured = false; // ass always needs to be reconfigured
```

Comment:
i saw the `reset()` above and am wondering: does it need to be there too?

Reply:
I don't believe so. The `reset` function seems to trigger when sub tracks are changed/reloaded/etc. (happens up in the core of mpv). These conditions don't require reconfiguring ass.